src/test/java/Pages/SearchPage.java

```diff
@@ -0,0 +1,45 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class SearchPage extends BasePage{
+
+    public SearchPage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+
+    //page Locators
+    By searchBar = By.cssSelector("input[type='search']");
+    By viewAllButton = By.cssSelector("button[data-test='view-all-songs-btn']");
+    By firstSong = By.cssSelector(".search-results .items tr:nth-child(1)");
+    By addToButton = By.cssSelector("button.btn-add-to");
+    By addToPlaylist = By.cssSelector("#songResultsWrapper .existing-playlists  li:nth-child(3)");
+    By successAlert = By.cssSelector(".alertify-logs .success");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```